README.md

```diff
@@ -16,7 +16,8 @@
 
 ## Live Demo 
 
-[liveDemoLink](https://elfin-git.github.io/Capstone-project-2/dist)
+[liveDemoLink](https://elfin-git.github.io/Capstone-project-2/dist/)
+You can click [here](https://drive.google.com/file/d/1vLHKtaT8Pp3Crx5S0LBR63rwnihQKqoE/view?usp=sharing) for the video presentation.
```

Comment:
- [ **OPTIONAL** ] Good job making your changes on your video link as right now it is readable. So, I noticed that according to the project's requirement your video should **contain a highlight about a piece of the code you find interesting or you are proud of making**. This could not be mentioned by the previous reviewer as he didn't watch your video.